File: Day-05 ( Validate ATM Cash Withdrawal )/Python/Solution.py
accountBalance, withdrawalAmount = input().split()

accountBalance = float(accountBalance)
withdrawalAmount = int(withdrawalAmount)

transactionFee = 2

valid = (withdrawalAmount > 0 and
         withdrawalAmount % 100 == 0 and
         accountBalance >= withdrawalAmount + transactionFee)

print("Transaction Valid:", str(valid).lower())

if valid:
    remainingBalance = accountBalance - withdrawalAmount - transactionFee
    print(f"Remaining Balance: {remainingBalance:.2f}")

File: Day-05 ( Validate ATM Cash Withdrawal )/CPP/Solution.cpp
#include <iostream>
#include <iomanip>
using namespace std;

int main() {
    double accountBalance;
    int withdrawalAmount;

    cin >> accountBalance >> withdrawalAmount;

    const double transactionFee = 2.0;

    bool valid = withdrawalAmount > 0 &&
                 withdrawalAmount % 100 == 0 &&
                 accountBalance >= withdrawalAmount + transactionFee;

    cout << boolalpha;
    cout << "Transaction Valid: " << valid << endl;

    if (valid) {
        double remainingBalance = accountBalance - withdrawalAmount - transactionFee;
        cout << fixed << setprecision(2);
        cout << "Remaining Balance: " << remainingBalance << endl;
    }

    return 0;
}



